terraform/services/api-rds/main.tf

```diff
@@ -144,7 +89,7 @@ resource "aws_db_parameter_group" "v16_parameter_group" {
   }
   parameter {
     name         = "rds.logical_replication"
-    value        = 0 # contains(["ab2d-dev", "ab2d-east-impl"], local.db_name) ? "1" : "0" # To support blue-green deployment for PostGres16 upgrade
+    value        = 0
     apply_method = "pending-reboot"
   }
 
```

Comment:
Why were the parameters that will be deleted removed? Are they necessary?

Reply:
If you're talking about the v15 group, I removed that per our conversation; if you're talking about the text past the # in line 147 (92 after changes) then basically I was leaving that as a shortcut if we needed it for blue/green deployments but since we're not doing those it's unnecessary clutter

Reply:
I mean in the v16 group, resulting in the parameters removed in the plan:

```
  # aws_db_parameter_group.v16_parameter_group will be updated in-place
  ~ resource "aws_db_parameter_group" "v16_parameter_group" {
        id           = "ab2d-dev-rds-parameter-group-v16"
        name         = "ab2d-dev-rds-parameter-group-v16"
        tags         = {}
        # (5 unchanged attributes hidden)

      - parameter {
          - apply_method = "pending-reboot" -> null
          - name         = "max_logical_replication_workers" -> null
          - value        = "22" -> null
        }
      - parameter {
          - apply_method = "pending-reboot" -> null
          - name         = "max_wal_senders" -> null
          - value        = "20" -> null
        }
      - parameter {
          - apply_method = "pending-reboot" -> null
          - name         = "max_worker_processes" -> null
          - value        = "25" -> null
        }
```

Reply:
Got it - This was a convo between me and @oluwolenpbc where he stated that with logical replication turned off as we're not doing blue-green deployments, these parameters are not needed moving forward.

Reply:
Thanks for explaining! I think this looks good then